server/internal/repositories/orm/models/scanner.go

```diff
@@ -0,0 +1,24 @@
+package models
+
+import (
+	"encoding/json"
+	"reflect"
+)
+
+// ScanJSON asserts src as []byte or string type and unmarshal to ptr.
+func ScanJSON(src, ptr any) error {
```

Comment:
@quocbang Since you use the `any ` type we can put either pointer of nil in the request you should write test cases for that and modify the function so that it also supports a pointer of src received

Reply:
@leducthai As far as I know, the Scan method supports only these types and it doesn't have support for pointer

![image](https://github.com/quocbang/data-flow-sync/assets/89030755/23b20739-9ec2-494f-aa3e-28c003070cfa)